stt_engines/deepspeech/uninstall.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+cd "$(dirname "${BASH_SOURCE[0]}")"
+source bin/activate
+
+sudo pip uninstall deepspeech
+rm deepspeech*
+rm -rf bin include lib share
+
+deactivate
+cd "$jv_dir"
```

Comment:
- cd into stt_engines/deepspeech and activate the virtualenv
- uninstall deepspeech using pip
- remove the model, scorer and logfile
- remove the directories created by virtualenv
- deactivate the virtualenv and cd into $jv_dir